index.js

```diff
@@ -33,10 +35,12 @@ function openConfig (cb) {
     return tryOpenConfig(path.join(os.tmpdir(), configfile), cb);
   }
 
-  tryOpenConfig(path.join(userHome, configfile), function (err, fd) {
-    if (err) return tryOpenConfig(path.join(os.tmpdir(), configfile), cb);
-    return cb(null, fd);
-  });
+  fs.mkdir(configPath, function () {
```

Comment:
@tkellen I understand but regardless of an error or not `tryOpenConfig` will be called so if the directory can not be created `os.tmpdir` will be used. Is that enough or would using `mkdirp` be better in some case?